classes/monograph/SubmissionFileDAO.inc.php

```diff
@@ -35,7 +35,7 @@ function __construct() {
 	/**
 	 * @copydoc PKPSubmissionFileDAO::fromRow()
 	 */
-	function fromRow($row) {
+	function fromRow($row, $fileImplementation = null) {
```

Comment:
@crism, I think it would probably be useful to note that `$fileImplementation` is a dead end in the function's self-docs.